dependencies.yaml

```diff
@@ -61,6 +62,9 @@ lint_requires:
   perl(Perl::Critic::Policy):
   perl(Perl::Critic::Utils):
 
+python_style_requires:
```

Comment:
Note that adding this simply to `devel_requires` would have the same effect but would be simpler.

Reply:
It's probably not necessary at all because `python_style_requires` is defined dynamically in the spec here:
`%if %{with black}`

Reply:
oh wait, sorry, to appear in the Dockerfile, this *is* actually needed

Reply:
@Martchus  `black` is unfortunately only packed in Tumbleweed so needs a small difference from standard `devel_requires`